cfy_manager/components/rabbitmq/rabbitmq.py

```diff
@@ -651,3 +654,32 @@ def _is_ipv6_enabled():
         ipv6_enabled = False
 
     return ipv6_enabled
+
+
+def _deploy_rebalancer_script_and_create_cronjob(self):
+    logger.info('Deploying queue rebalancing script...')
+    source_path = join(constants.COMPONENTS_DIR,
+                       RABBITMQ,
+                       SCRIPTS,
+                       'rebalance_queues.py')
+    files.deploy(source_path, QUEUES_REBALANCING_SCRIPT_PATH)
+    common.chmod('+x', QUEUES_REBALANCING_SCRIPT_PATH)
+    common.chown(constants.CLOUDIFY_USER,
+                 constants.CLOUDIFY_GROUP,
+                 QUEUES_REBALANCING_SCRIPT_PATH)
+
+    logger.info('Creating cron job for rebalancing queues...')
+    random_minute = randint(0, 59)
+    time_string = '{0} */4 * * *'.format(random_minute)    # run every 4 hours
```

Comment:
No need for a random minute- just do it on the hour, since you already have a check elsewhere to make sure it doesn't run on more than one at once.